client/src/components/NavBar.tsx

```diff
@@ -31,7 +43,7 @@ function NavBar() {
         <li>
           <NavLink to={"/contact"} className="NavLink-nav">
             {" "}
-            <h3 className="h3-text">Nous contacter</h3>
+            <h3 id="contacter">Nous Contacter</h3>
```

Comment:
Les `id` ne devraient avoir pour valeur que des termes en anglais.
Ici, par exemple, `contact-title` devrait être préféré.

Reply:
is it made a leader!!